airflow/example_dags/tutorial_taskflow_api_etl.py

```diff
@@ -45,22 +35,23 @@
 # [END default_args]
 
 # [START instantiate_dag]
-with DAG(
-    'tutorial_taskflow_api_etl_dag',
-    default_args=default_args,
-    description='TaskFlow API ETL DAG tutorial',
-    schedule_interval=None,
-    start_date=days_ago(2),
-    tags=['example'],
-) as dag:
-    # [END instantiate_dag]
-
+@dag(default_args=default_args, schedule_interval=None, start_date=days_ago(2))
+# [END instantiate_dag]
+def tutorial_taskflow_api_etl():
     # [START documentation]
```

Comment:
We'll need to see how this renders in context -- we may need to make it clearer what this does